lib/node_modules/@stdlib/stats/base/meanpn/lib/main.js

```diff
@@ -1,7 +1,7 @@
 /**
 * @license Apache-2.0
 *
-* Copyright (c) 2020 The Stdlib Authors.
+* Copyright (c) 2025 The Stdlib Authors.
```

Comment:
```suggestion
* Copyright (c) 2020 The Stdlib Authors.
```

You didn't touch this file, so why are you updating the copyright year?